lib/mongoidable/instance_abilities.rb

```diff
@@ -23,5 +25,7 @@ def own_abilities
       end
       own_abilities
     end
+
+    memoize :own_abilities
```

Comment:
`memoizing` `own_abilities` in this file seems appropriate, but at odds with how it is called [here](https://github.com/CardTapp/mongoidable/pull/137/files#diff-3dc574e58966ab4db7b92c6937ce4679f407718362049a2f88867d7760c28cafR27)

Reply:
https://github.com/CardTapp/mongoidable/pull/137#discussion_r694327505